nemo/collections/llm/gpt/model/base.py

```diff
@@ -32,6 +32,14 @@
 
 _, HAVE_TE = safe_import("transformer_engine")
 
+# Gradient accumulation fusion may be enabled if available, for more information see:
+# https://github.com/NVIDIA/Megatron-LM/blob/01945b98d1ea3a2acb5e8301e181a328104f4856/megatron/core/tensor_parallel/layers.py#L575
+_grad_accum_fusion_available = True
+try:
+    import fused_weight_gradient_mlp_cuda
+except ImportError:
+    _grad_accum_fusion_available = False
```

Comment:
np: Consider adding a logline how to install the fused_weight_gradient_mlp_cuda. 

(I see it makes from Apex and is optional)